MAINTAINERS

```diff
@@ -50,6 +50,7 @@
 "Sai Chaithanya",@mittachaitu,MayaData #control-plane-maintainers
 "Sebastien Dionne",@survivant,Jerabi inc. #control-plane-maintainers
 "Shubham Bajpai",@shubham14bajpai,MayaData #control-plane-maintainers
+"Sjors Gielen",@sgielen,CCV Group #control-plane-maintainers, cstor-engine-maintainers
```

Comment:
Could you change my organization to Independent? Just to prevent any accidental implication from my employer :-)